### PR TITLE
Added Italian localization

### PR DESCRIPTION
**Pull request by Knocks83**
Added the Italian language

### CODE REVIEW

**Comment by NicoAiko** [comment]
Thanks for your contribution!
I'll take a quick look and then come back with another comment.

File: src/assets/locales/it.json [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+    }
+  },
+  "misc": {
+    "aniList": {
+      "listStatusses": {
+        "completed": "Completato",
+        "dropped": "Abbandonato",
+        "paused": "In pausa",
+        "planning": "Pianificati",
+        "In ripetizione": "Repeating",
```
Comment (NicoAiko) [inline src/assets/locales/it.json]: The key has been changed here instead of the value

File: src/assets/locales/it.json [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+        "fall": "Autunno",
+        "autumn": "Autunno",
+        "winter": "Inverno"
+      }
+    },
+    "dates": {
+      "full": "Do MMMM YYYY",
+      "monthAndYear": "MMMM YYYY",
+      "yearOnly": "YYYY",
+      "dateUnknown": "Date unknown"
```
Comment (NicoAiko) [inline src/assets/locales/it.json]: This value is still English.

File: src/assets/locales/it.json [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+          "readChapter": "Capitoli letti {1} of {0}",
+          "plansToRead": "Pianificati da leggere {0}",
+          "rereadChapter": "Capitoli riletti {1} of {0}",
+          "pausedReading": "Letture in pausa {0}"
+        },
+        "howToAuthenticate": {
+          "header": "Come accedere:",
+          "text": [
+            "Vai nella pagina Impostazioni (angolo in alto a destra) e quindi vai nel tab \"AniList\".",
+            "Qua troverai un bottone \"Login\" che aprirà una finestra di accesso di AniList.",
```
Comment (NicoAiko) [inline src/assets/locales/it.json]: The action of login seems to be `Accedi`, not `Login`

File: src/assets/locales/it.json [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+    "sort": "Ordina e Filtra",
+    "toPageTop": "Torna in cima"
+  },
+  "alerts": {
+    "adultContent": "Questo media è considerato per adulti.",
+    "noDescription": "Descrizione non disponibile.",
+    "noEnglishTitle": "Titolo in inglese non disponibile.",
+    "noInformation": "Nessuna informazione disponibile.",
+    "noSynonyms": "Nessun sinonimo disponibile.",
+    "notYetInList": "This media is not yet in your list.",
```
Comment (NicoAiko) [inline src/assets/locales/it.json]: This value is still English

File: src/assets/locales/it.json (hunk 3 above)
Comment (nattadasu) [reply src/assets/locales/it.json]: Maybe in some cases, they use English word for casual usage? :thinking:
Like some words in Indonesian were "translated" from borrowing some English words then the words would be *Indonesianized* (Like `Export` to `Ekspor`)
(That's my thought, tho)

Comment (NicoAiko) [reply src/assets/locales/it.json]: Well, I'd agree on this if the translated `Login` in `actions.login` wouldn't be `Accedi`.

For consistency reasons, the button should be called the same way as the translation that provides the explanation.

Calling the login button in two different ways would confuse users.

Comment (Knocks83) [reply src/assets/locales/it.json]: Yes, some words usually don't get translated but I think I've already translated it so I'll change this too

**Comment by Knocks83** [comment]
I should have fixed everything, if you find anything wrong just comment and I'll fix right away.

File: src/assets/locales/it.json (hunk 4 above)
Comment (NicoAiko) [reply src/assets/locales/it.json]: @Knocks83 I think this one you missed.

**Comment by Knocks83** [comment]
Fixed!

**Pull request merged**